spring-boot-configuration/README.md

```diff
@@ -0,0 +1,7 @@
+# Spring Boot Configuration
+
+This module contains articles about Spring Boot Configuration.
+
+### Relevant Articles: 
+
+- [DataJpaTest, Unable to Find Spring Boot Configuration](https://www.baeldung.com/data-jpa-test-unable-to-find/)
```

Comment:
let's remove it - editors add that after the article is published, as we don't know the link yet.

Reply:
Okay, learned something new :-)

Updated the PR :-)